samples/_opengl/ObjLoader/src/ObjLoaderApp.cpp

```diff
@@ -93,7 +93,7 @@ void ObjLoaderApp::writeObj()
 	fs::path filePath = getSaveFilePath();
 	if( ! filePath.empty() ) {
 		console() << "writing mesh to file path: " << filePath << std::endl;
-		writeObj( writeFile( filePath ), mMesh );
+		ci::writeObj( writeFile( filePath ), mMesh );
```

Comment:
Some minor nits following: `ci::` prefix isn't necessary, we generally don't use these in samples and instead put a `using namespace ci` at the top.